src/org/zaproxy/zap/extension/api/OptionsParamApi.java

```diff
@@ -68,11 +76,15 @@ public OptionsParamApi() {
     protected void parse() {
         
 		enabled = getBooleanFromConfig(ENABLED, true);
+		uiEnabled = getBooleanFromConfig(UI_ENABLED, true);
 		secureOnly = getBooleanFromConfig(SECURE_ONLY, false);
 		disableKey = getBooleanFromConfig(DISABLE_KEY, false);
 		incErrorDetails = getBooleanFromConfig(INC_ERROR_DETAILS, false);
 		autofillKey = getBooleanFromConfig(AUTOFILL_KEY, false);
 		enableJSONP = getBooleanFromConfig(ENABLE_JSONP, false);
+		noKeyForSafeOps = getBooleanFromConfig(NO_KEY_FOR_SAFE_OPS, false);
+		reportPermErrors = getBooleanFromConfig(REPORT_PERM_ERRORS, false);
+		nonceTimeToLiveInSecs = getIntFromConfig(NONCE_TTL_IN_SECS, 5 * 60);	// 5 mins

```

Comment:
` 5 * 60);	// 5 mins` seems to be used multiple times, might be better as a constant?